dask/dataframe/core.py

```diff
@@ -8441,7 +8446,7 @@ def _convert_to_numeric(series, skipna):
     return series.view("i8").mask(series.isnull(), np.nan)
 
 
-def _sqrt_and_convert_to_timedelta(partition, axis, *args, **kwargs):
+def _sqrt_and_convert_to_timedelta(partition, axis, dtype=None, *args, **kwargs):
```

Comment:
This doesn't always convert to `timedelta` anymore, is that right?

Reply:
Actually it casts back to timedelta now which it didn’t do before in NaT cases

Reply:
Oic, that's why you added explicit `dtype`. 👍